modules/claims_api/lib/claims_api/evidence_waiver_pdf/pdf.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class EvidenceWaiver
+    # Example call to use this tool:
+    #
+    # waiver = ClaimsApi::EvidenceWaiver.new(target_veteran: target_veteran)
+    # pdf = waiver.construct(response: params[:response])
+    def initialize(target_veteran:)
+      @page1_path = nil
+      @target_veteran = target_veteran
+    end
+
+    def construct(response: true)
+      fill_pdf(response)
+    end
+
+    protected
+
+    # @return [String] Path to page 1 pdf template file
+    def page1_template_path
+      Rails.root.join('modules', 'claims_api', 'config', 'pdf_templates', '5103', '1.pdf')
+    end
+
+    def signature
+      name = [@target_veteran.first_name, @target_veteran.middle_name, @target_veteran.last_name].compact.join(' ')
+      "#{name[0...180]}\n- By digital auth to api.va.gov"
```

Comment:
did we check on if there would be any concerns using `By digital auth to api.va.gov` vs the tickets `Signed by digital authentication to api.va.gov`?